app/views/pages/_search.html.erb

```diff
@@ -1,20 +1,9 @@
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <h1 class="heading-xlarge">
-      <%= t('.find_gov_data') %>
-    </h1>
-    <p class="lede">
-    <%= t('.lede') %>
-    </p>
-  </div>
-</div>
-<div class="grid-row">
-  <div class="column-two-thirds dgu-search-box">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds dgu-search-box">
     <form action="/search" method="GET">
```

Comment:
Slightly picky - you could remove a `div` element by adding `class="govuk-grid-column-two-thirds dgu-search-box"` to this form element.
```suggestion
    <form action="/search" method="GET" class="govuk-grid-column-two-thirds dgu-search-box">
```